### DISCUSSION TITLE
miaomiao3 and libre2 on android wearOS with BT collector

### DISCUSSION THREAD

**Discussion by cr-ellis** [NightscoutFoundation/xDrip#2427] [General]
I currently have xdrip+ working great with libre2 (UK) through a miaomiao3.  It's getting started through a libre reader and then connected to xdrip on android phone.  This is working fine.

I have a strong interest in getting a smartwatch working fully with the miaomiao.  Preliminary attempts with a ticwatch and the OOP2 apk installed gives data errors.  It looks in the code like only libre1 is supported in wear.

I'm reaching out to ask if anyone is currently working on this.  I'm very willing to put some developer time to get this to work.  It looks like the libre2 code for the app could be adapted for wear also.

**Comment by SimonPhilpott** [comment]
I'm also very interested in this, it's frustraiting going backwards from a very reliable setup on my TIcWatch Pro 3 running as a collector via MiaoMiao 1 with Libre 1. To now having to switch back to using my phone as the collector due to OOP2.

**Reply by cr-ellis** [reply]
There is a pull request open for this.  Maybe you'd like to help test it?

https://github.com/NightscoutFoundation/xDrip/pull/2654

**Reply by SimonPhilpott** [reply]
@cr-ellis  Absolutely, I'll give it a go first thing tomorrow so I've got a full day to test

**Comment by Navid200** [comment]
@psonnera
